### DISCUSSION TITLE
playercorefactory help

### DISCUSSION THREAD

**Discussion by derrgoo** [abskmj/notes#30] [Q&A]
Thank you for your example. This helped me allot. However, I want it to only play my PVR live tv because internal Kodi player has a problem playing and always dropping. Therefore, I am really needing help on a choice of internal Kodi player for everything else but IPTV PVR. Is there a way to says play using internal or VLC ? IF so, what lines do I need to add?

**Comment by abskmj** [comment]
I'm not sure of the exact changes that would work for you but rules in the files allow setting players based on protocol or extension. You could set accordingly.
